moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -469,13 +457,6 @@ const ProjectView = () => {
                             horizontal: "center",
                           }}
                         >
-                          <KnackSync
-                            project={data.moped_project[0]}
-                            closeHandler={handleMenuClose}
-                            snackbarHandler={handleSnackbarOpen}
-                            refetch={refetch}
-                          />
-
```

Comment:
All the changes in this file are the removal of the old Knack Sync option in the drop-down menu for projects.